tests/Type/ItemPriceTest.php

```diff
@@ -1,23 +1,30 @@
 <?php
+namespace Blesta\Pricing\Tests\Unit\Type;
```

Comment:
Namespace should be `Blesta\Pricing\Tests\Type;` *OR* test should be moved into a `Unit` directory within `tests/` (notice case).